cli/config/configuration-fc.toml.in

```diff
@@ -315,6 +315,12 @@ disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 # (default: false)
 #disable_new_netns = true
 
+# if enable, the runtime use the parent cgroup of a container PodSandbox.  This
```

Comment:
We *really* need to create a bunch of TOML fragments real soon now (TM) to avoid having to update all these config files ;(